src/frontend/js/components/CourseRunEnrollment/index.tsx

```diff
@@ -233,12 +235,23 @@ export const CourseRunEnrollment: React.FC<CourseRunEnrollmentProps & CommonData
     case step === Step.ENROLLED:
       return (
         <React.Fragment>
-          <a href={courseRun.resource_link} className="course-run-enrollment__cta">
-            <FormattedMessage {...messages.goToCourse} />
-          </a>
-          <div className="course-run-enrollment__helptext">
-            <FormattedMessage {...messages.enrolled} />
-          </div>
+          {courseRun.starts_in_message ? (
+            <div>
+              <a href={courseRun.dashboard_link} className="course-run-enrollment__cta">
+                <FormattedMessage {...messages.enrolled} />
+              </a>
```

Comment:
If `dashboard_link` is null, we could not wrap `<FormattedMessage />` into a link.